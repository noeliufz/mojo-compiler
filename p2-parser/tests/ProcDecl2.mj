proc foo(i: int) { i = i + 1; }
