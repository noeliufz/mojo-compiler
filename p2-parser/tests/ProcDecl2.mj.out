proc foo (
    val i: int;
  )
{
  i = i + 1;
}
